### ISSUE TITLE
sphinx-apidoc does not find .pyd files

### ISSUE THREAD

**Issue by deathride007** [sphinx-doc/sphinx#5728]
sphinx-apidoc does not include ".pyd" files which are compiled python modules from c++.
Related to issue #2546

This issue is caused by the list of PY_SUFFIXES, which currently only contains .py and .pyx files.
When adding '.pyd' to the PY_SUFFIXES, sphinx-apidoc is able to generate correct rst files.

Fix in /sphinx/ext/apidoc.py (line 53)
`PY_SUFFIXES = set(['.py', '.pyx', '.pyd'])`

**Comment by tk0miya** [comment]
@shimizukawa Could you check this please? Unfortunately, I'd never seen `.pyd` file... I need your help.

**Comment by shimizukawa** [comment]
This issue is resolved by https://github.com/sphinx-doc/sphinx/pull/6801 that has been shipped since Sphinx-2.3.0.

[Release 2.3.0 (released Dec 15, 2019)](https://www.sphinx-doc.org/en/master/changes.html#release-2-3-0-released-dec-15-2019)
> #2546: apidoc: .so file support
`.pyd` also supported on Windows.

```
(.venv) C:\>sphinx-build --version
sphinx-build 2.4.2

(.venv) C:\>python -c "print(__import__('sphinx.ext.apidoc').ext.apidoc.PY_SUFFIXES)"
('.py', '.pyx', '.cp38-win_amd64.pyd', '.pyd')
```
